Exec/mhd_tests/FastRarefaction/problem_initialize_mhd_data.H

```diff
@@ -0,0 +1,99 @@
+#ifndef problem_initialize_mhd_data_H
+#define problem_initialize_mhd_data_H
+
+AMREX_GPU_HOST_DEVICE AMREX_INLINE
+void problem_initialize_mhd_data (int i, int j, int k,
+                                  Array4<Real> const& B, const int ibdir,
+                                  const GeometryData& geomdata)
+{
+    int coord_type = geomdata.Coord();
+
+    const Real* dx = geomdata.CellSize();
+    const Real* problo = geomdata.ProbLo();
+
+    Real x = problo[0] + dx[0] * (static_cast<Real>(i) + 0.5_rt);
+
+    Real y = 0.0;
+#if AMREX_SPACEDIM >= 2
+    y = problo[1] + dx[1] * (static_cast<Real>(j) + 0.5_rt);
+#endif
+
+    Real z = 0.0;
+#if AMREX_SPACEDIM == 3
+    z = problo[2] + dx[2] * (static_cast<Real>(k) + 0.5_rt);
+#endif
+
+
+    if (problem::idir == 1) {
+        // initialize along the x-direction
+
+        if (ibdir == 0) {
+            if (x <= problem::split[0]+1) {
```

Comment:
Can you explain where the +1 comes from here?

Reply:
oh, I think that is a copy-paste from a different problem
I'll fix it

Reply:
oh wait... indeed that was in the mag intiialization originally, but not in the hydro state.  I am not sure why that is there.

Reply:
it looks like it is only in the problem direction that we do that.

Reply:
yeah, DaiWoodward did that too, that's weird but whatever.